src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -60,6 +60,13 @@ public int MaximumFollowRelLink
             set { base._maximumFollowRelLink = value; }
         }
 
+        /// <summary>
+        /// Gets or sets the ResponseHeadersVariable property.
+        /// </summary>
+        [Parameter]
+        [Alias("HV")]
```

Comment:
Maybe 'RHV'?

Reply:
I'm up for suggestions on this one. I was following suit with `SessionVariable`'s `SV`. I would have used `-HeadersVariable` for the name if that wouldn't have been mistaken for `-Headers`

Reply:
I prefer RHV too.

Reply:
updated